test/libsolidity/syntaxTests/pragma/invalid_range_operator_range_first.sol

```diff
@@ -0,0 +1,3 @@
+pragma solidity <0.8.17 - 0.8.20;
+// ----
+// ParserError 1684: (0-33): Invalid version pragma. You cannot use operators (<, <=, >=, >, ^) with version ranges (-).
```

Comment:
Please configure your editor to ensure that there is always a newline at the end of the file. Makes for less random noise in diffs.